docs/Private-IP-Configuration/README.md

```diff
@@ -0,0 +1,83 @@
+4. For the “Router” drop-down, EITHER select pre-created router OR click on “create new router”.
+<br> a. Complete router creation by providing name, description & region. Refer to GCP’s [Create-Router](https://cloud.google.com/network-connectivity/docs/router/how-to/create-router-vpc-on-premises-network) documentation for more details.
+5. Complete NAT gateway creation by providing required details. Refer to GCP’s [Create-NAT-Gateway](https://cloud.google.com/nat/docs/set-up-manage-network-address-translation) documentation for more details.
+
+![Router Details](https://user-images.githubusercontent.com/101806311/176452881-a8070ccc-0349-4459-a7ec-f33bff6b592e.png?raw=true "Router Details")
+
+![NAT Gateways](https://user-images.githubusercontent.com/101806311/176453003-da5e8e80-7acc-425c-a527-34ae23a75b95.png?raw=true "NAT Gateways")
+
+## Sample commands to trigger dataflow pipeline execution using above options
+
+Following section showcases how VPC-parameters can be given as CLI inputs to weather-mv dataflow pipeline.
+
+| Python                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | 
+|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| weather-mv --uris "gs://<font color="brown">$STORAGE_BUCKET</font>/*.nc"<br/>--output_table "<font color="brown">$HOST_PROJECT_ID</font>.<font color="brown">$DATASET_ID</font>.<font color="brown">$TABLE_ID</font>"<br/>--temp_location "gs://<font color="brown">$STORAGE_BUCKET</font>/tmp"<br/>--runner DataflowRunner<br/>--project <font color="brown">$HOST_PROJECT_ID</font><br/>--region <font color="brown">$REGION_NAME</font><br/>--no_use_public_ips <br/>--network=<font color="brown">$NETWORK_NAME</font><br/>--subnetwork=regions/<font color="brown">$REGION_NAME</font>/subnetworks/<font color="brown">$SUBNETWORK_NAME</font><br/> | 
```

Comment:
This renders really weird on the final page. Can you remove the `<font>` tags, etc. and just use a codeblock? i.e. via 3 backtics?

Reply:
Said changes implemented.